src/components/accessories/patientDataForm/PatientDataForm.tsx

```diff
@@ -98,6 +99,50 @@ const PatientDataForm: FunctionComponent<TProps> = ({
     }))
   );
 
+  const allAgeTypes = useSelector(
+    (state: IState) => state.ageTypes.getAllAgeTypes.data
+  );
+
+  const getBirthDateAndAge = (
```

Comment:
Can we have an hook for this?

Reply:
I'm not getting you well. Do you suggest to create an hook like the existing one `useCityOptions` ?

Reply:
Also, the function `getBirthDateAndAge()` is only used in `PatientDataForm` Component.